### PR TITLE
Implement small tax functions

### LINKED ISSUE #111: Add IITAX

### LINKED ISSUE #117: Add LumpSumTax

### LINKED ISSUE #118: Add ExpandIncome

### LINKED ISSUE #119: Add AfterTaxIncome

### PR DESCRIPTION
**Pull request by nikhilwoodruff**
Fixes #119 
Fixes #111 
Fixes #118 
Fixes #117 (assuming we should skip, as this seems to be a head tax that isn't in the baseline)

### CODE REVIEW

File: openfisca_us/parameters/tax/credits/child_tax_credit/refundable.yaml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+description: Whether the child tax credit is fully refundable
```
Comment (MaxGhenis) [inline openfisca_us/parameters/tax/credits/child_tax_credit/refundable.yaml:1]:
```suggestion
description: Whether the Child Tax Credit is fully refundable
```

File: openfisca_us/variables/finance/tax/outputs.py [hunk 2]
```diff
@@ -188,13 +194,47 @@ class expanded_income(Variable):
         """Broad income measure that includes benefit_value_total"""
     )
 
+    def formula(tax_unit, period, parameters):
```
Comment (MaxGhenis) [inline openfisca_us/variables/finance/tax/outputs.py:197]: Expanded income isn't used for tax calculations, but it is what taxcalc uses as their income measure for binning into deciles. Should we include it?

File: openfisca_us/variables/finance/tax/outputs.py [hunk 3]
```diff
@@ -944,3 +994,22 @@ class nontaxable_ubi(Variable):
     entity = TaxUnit
     definition_period = YEAR
     documentation = """Amount of UBI benefit excluded from AGI"""
+
+
+class aftertax_income(Variable):
```
Comment (MaxGhenis) [inline openfisca_us/variables/finance/tax/outputs.py:999]: similarly, aftertax_income and benefit_value_total aren't used for calculations. aftertax_income is the net income without benefits, and therefore used for summarizing tax reforms. Could be important for testing for now but I think eventually at least we may want to remove it.

File: openfisca_us/variables/finance/tax/outputs.py [hunk 4]
```diff
@@ -944,3 +994,22 @@ class nontaxable_ubi(Variable):
...
+    value_type = float
+    entity = TaxUnit
+    label = "After-tax income"
+    definition_period = YEAR
+
+    def formula(tax_unit, period, parameters):
+        expanded = tax_unit("expanded_income", period)
+        combined = tax_unit("combined", period)
```
Comment (MaxGhenis) [inline openfisca_us/variables/finance/tax/outputs.py:1007]:
```suggestion
        combined_tax = tax_unit("combined", period)
```

**Review by MaxGhenis — APPROVED** [review]

File: openfisca_us/variables/finance/tax/outputs.py (hunk 2 above)
Comment (nikhilwoodruff) [reply openfisca_us/variables/finance/tax/outputs.py:197]: I can see benefit in including it (even if it's redundant/unused) for validation later on

File: openfisca_us/variables/finance/tax/outputs.py (hunk 3 above)
Comment (nikhilwoodruff) [reply openfisca_us/variables/finance/tax/outputs.py:999]: Yeah I agree, but I think it might still be helpful when validating the model

**Pull request merged**